sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsDiskStore.scala

```diff
@@ -36,31 +36,37 @@ class RapidsDiskStore(
   override def createBuffer(
       incoming: RapidsBuffer,
       stream: Cuda.Stream): RapidsBufferBase = {
-    val incomingBuffer = incoming.getMemoryBuffer
-    try {
-      val hostBuffer = incomingBuffer match {
-        case h: HostMemoryBuffer => h
-        case _ => throw new UnsupportedOperationException("buffer without host memory")
-      }
-      val id = incoming.id
-      val path = if (id.canShareDiskPaths) {
-        sharedBufferFiles.computeIfAbsent(id, _ => id.getDiskPath(diskBlockManager))
-      } else {
-        id.getDiskPath(diskBlockManager)
-      }
-      val fileOffset = if (id.canShareDiskPaths) {
-        // only one writer at a time for now when using shared files
-        path.synchronized {
-          copyBufferToPath(hostBuffer, path, append = true)
+    if (incoming.alreadySpilledToDisk) {
+      logDebug(s"Respilling $incoming ${incoming.id} of size ${incoming.size} to disk")
```

Comment:
nit: Could we update this because we didn't actually move any bytes to spill, and it might be good that it was more obvious in the log.

Reply:
Done.